unstoppable-ios-app/domains-manager-ios/Entities/Wallets/UDWallet+Signing.swift

```diff
@@ -19,6 +19,13 @@ extension UDWallet {
         return walletName.contains("alpha")
     }
     
+    var shouldEncodeHexMessage: Bool {
+        guard let walletName = self.getExternalWalletName()?.lowercased() else {
+            return false
+        }
+        return walletName.contains("meta") || walletName.contains("rain") || walletName.contains("okx")
```

Comment:
Wouldn't it be better to get walletConnectionInfo?.externalWallet?.make and switch?